products/rhel9/profiles/e8.profile

```diff
@@ -107,7 +107,6 @@ selections:
   - audit_rules_login_events_tallylog
   - audit_rules_login_events_faillock
   - audit_rules_login_events_lastlog
-  - audit_rules_login_events
```

Comment:
Please add the removed rules to the RHEL 9 default profile so that they will be kept in the built RHEL 9 data stream.